zuul/main.yaml

```diff
@@ -38,6 +38,8 @@
           - opentelekomcloud/ui-driver-otc
           - opentelekomcloud/ui-cluster-driver-otc
           - opentelekomcloud/vault-plugin-secrets-openstack
+          ### OpenTelekomCloud Blueprints
```

Comment:
we have blueprints org, and this comment may be misleading
what do you think?

Reply:
this is a similar separation block to the one 2 lines above - it splits across orgs